src/components/Atoms/Button/Button.css

```diff
@@ -39,7 +39,9 @@
 /* Inherits button always */
 .btn,
 .btnWhite,
-.btnOrange {
+.btnOrange,
+.btnDark,
+.btnBlue {
   @mixin btn;
 }
 
```

Comment:
The button has too much padding for this context. 

**Adjust these:**
line-height: 1.5;
padding: 0 0.3rem;
height: 24px;

Reply:
Fixed. Left out `height` since it should not be used here in combination with `line-height`.